.ddev/.gitignore

```diff
@@ -23,11 +23,14 @@
 /db_snapshots
 /import-db
 /import.yaml
-/mutagen
+/mutagen/mutagen.yml
```

Comment:
@anvmn This file is managed by ddev (as you can see in the first lines of the file), but it seems it is committed to github in early version. I think the proper fix is to 

```
git rm .ddev/.gitignore
git push ...
```